openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/EditScreenDialog.vue

```diff
@@ -193,10 +219,26 @@ export default {
     this.editor.container.remove()
   },
   methods: {
+    showContextMenu: function (event) {
+      this.menuX = event.pageX
+      this.menuY = event.pageY
+      this.docsKeyword = this.editor.getSelectedText()
```

Comment:
This seems really fragile.  Shouldn't it find the first word of the line and select that.  What if nothing is selected and I just right click on the word?